src/panels/lovelace/views/hui-view.ts

```diff
@@ -53,6 +57,7 @@ declare global {
     "ll-create-card": HASSDomEvent<HASSDomEvents["ll-create-card"]>;
     "ll-edit-card": HASSDomEvent<HASSDomEvents["ll-edit-card"]>;
     "ll-delete-card": HASSDomEvent<HASSDomEvents["ll-delete-card"]>;
+    "ll-duplicate-card": HASSDomEvent<HASSDomEvents["ll-duplicate-card"]>;
```

Comment:
shouldn't there be `ll-copy-card` here as well?

Reply:
Yep, I missed that one 😅